creator-node/compose/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       - manualSyncsDisabled=${manualSyncsDisabled}
       - devMode=${DEV_MODE}
       - spOwnerWallet=${spOwnerWallet}
-      - openRestyCacheCIDEnabled=${openRestyCacheCIDEnabled}
+      - contentCacheLayerEnabled=${contentCacheLayerEnabled}
```

Comment:
any issues with this name change affecting back-compat of rollout?

Reply:
this variable isnt used anywhere since we disabled the cache layer, so this name change shouldnt impact anything

Reply:
wasn't it enabled on SPs before we disabled it tho?

Reply:
I think this change is fine. if we roll out this env var name change, the config will default the value to false if it is not present.

Reply:
sg!